tests/test_aggregated.py

```diff
@@ -516,7 +528,10 @@ def test_aggregate_obsm_labels():
     )
 
     expected = ad.AnnData(
-        obs=pd.DataFrame({"labels": pd.Categorical(list("abc"))}, index=list("abc")),
+        obs=pd.DataFrame(
+            {"labels": pd.Categorical(list("abc")), "n_obs_aggregated": [5, 3, 4]},
+            index=list("abc"),
+        ),
```

Comment:
Construct from `label_counts` variable please!
```suggestion
        obs=pd.DataFrame(
            {"labels": pd.Categorical([lc[0] for lc in label_counts]), "n_obs_aggregated": [lc[1] for lc in label_counts]},
            index=[lc[0] for lc in label_counts],
        ),
```